third_party/gems/bundle.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# --- begin runfiles.bash initialization ---
+# Copy-pasted from Bazel's Bash runfiles library (tools/bash/runfiles/runfiles.bash).
+set -euo pipefail
+if [[ ! -d "${RUNFILES_DIR:-/dev/null}" && ! -f "${RUNFILES_MANIFEST_FILE:-/dev/null}" ]]; then
```

Comment:
rather than being copy-pasted into every script, can it be reused? 
can we `source` something?

Reply:
Unfortunately, this is what sets up `rlocation` so that we can reliably find artifacts from `data` attributes.

Reply:
(Copy-pasting that snippet is the official recommendation from bazel: https://github.com/bazelbuild/bazel/blob/master/tools/bash/runfiles/runfiles.bash)